core/controllers/topics_and_skills_dashboard.py

```diff
@@ -60,12 +60,21 @@ def get(self):
         skill_summary_dicts = [
             summary.to_dict() for summary in skill_summaries]
 
+        triaged_skill_ids = topic_services.get_all_triaged_skills()
```

Comment:
Should be skill_ids not skills (in method name)

Also maybe get_all_skill_ids_assigned_to_some_topic() or get_all_categorized_skill_ids() is clearer. I wonder whether we should also change untriaged --> uncategorized in order to avoid introducing a new term of art. What do you think?

Reply:
We are already using uncategorized to refer to those skill not part of any subtopic, in a topic, right? Not sure, if we want to name those not in any topic also as uncategorized.

Reply:
Done.